tecs/tecs.cpp

```diff
@@ -361,12 +361,12 @@ void TECS::_update_throttle_setpoint(const float throttle_cruise, const matrix::
 		if (_integrator_gain > 0.0f) {
 			// Calculate throttle integrator state upper and lower limits with allowance for
 			// 10% throttle saturation to accommodate noise on the demand.
-			float integ_state_max = _throttle_setpoint_max - _throttle_setpoint + 0.1f;
-			float integ_state_min = _throttle_setpoint_min - _throttle_setpoint - 0.1f;
+			const float integ_state_max = _throttle_setpoint_max - _throttle_setpoint + 0.1f;
+			const float integ_state_min = _throttle_setpoint_min - _throttle_setpoint - 0.1f;
 
 			// Calculate a throttle demand from the integrated total energy error
```

Comment:
I would like to see the term  'specific total energy' used consistently in the comments.